src/Core/InfoBar/IInfoBar.cs

```diff
@@ -42,4 +42,14 @@ public interface IInfoBar
         /// </summary>
         void Close();
     }
+
+    public class InfoBarButtonClickedEventArgs : EventArgs
+    {
+        public string ClickedButtonText { get; }
```

Comment:
From the original event we have access to IVsInfoBarActionItem but:
1) it's a VS object and this is Core
2) It doesn't actually have anything useful in it besides the text

https://docs.microsoft.com/en-us/dotnet/api/microsoft.visualstudio.shell.interop.ivsinfobaractionitem?view=visualstudiosdk-2022

Reply:
The ActionContext might be useful as a way of passing arbitrary context data, but we don't need it now.